boa/src/symbol/mod.rs

```diff
@@ -42,65 +55,73 @@ pub struct WellKnownSymbols {
     unscopables: RcSymbol,
 }
 
+const RESERVED_SYMBOL_HASHES: u64 = 128;
+
+thread_local! {
+    /// Cached well known symbols
+    static WELL_KNOW_SYMBOLS: WellKnownSymbols = WellKnownSymbols::new();
+
+    /// Symbol hash.
+    ///
+    /// For now this is an incremented u64 number.
+    static SYMBOL_HASH_COUNT: Cell<u64> = Cell::new(RESERVED_SYMBOL_HASHES);
+}
+
 impl WellKnownSymbols {
-    pub(crate) fn new() -> (Self, u64) {
+    fn new() -> Self {
```

Comment:
> Maybe we should change the `new()` function and be the more idiomatic `default()` with the `Default` trait?

In this case, `new` is better since using `Default` would make it public and there should only be one instance of well known symbols.